packages/node_modules/@node-red/editor-client/src/sass/tab-config.scss

```diff
@@ -54,6 +54,9 @@ ul.red-ui-sidebar-node-config-list {
         white-space: nowrap;
         overflow: hidden;
         text-overflow: ellipsis;
+        &:not(:last-child) {
```

Comment:
@ralphwetzel, out of curiosity, why did you add `&:not(:last-child)`?

Reply:
This adaptation of width is only necessary if the label shares its parent space with another element.
Do you think, this constraint is not necessary?

Reply:
I don't think so. Unless I'm missing something